yb-voyager/src/namereg/namereg.go

```diff
@@ -345,8 +374,12 @@ func (reg *NameRegistry) LookupTableName(tableNameArg string) (sqlname.NameTuple
 				}
 			}
 			if err != nil {
-				// `err` can be: no default schema, no matching name, multiple matching names.
-				return sqlname.NameTuple{}, fmt.Errorf("lookup source table name [%s.%s]: %w", schemaName, tableName, err)
+				if ignoreIfOtherSideMappingNotFound {
```

Comment:
how do we know for sure that the error is "not found"
Technically, it could be because of other scenarios as well, right? Ideally, we should be sure that the error we are ignoring is "not found" error. 

There seems to be an `ErrNameNotFound` error. We can check for that specifically.

Reply:
Yeah, makes sense we should ignore the other errors. So just ignoring the `ErrNameNotFound` type of error only.